### DISCUSSION TITLE
Pause at beginning of new layer

### DISCUSSION THREAD

**Discussion by Scotshane** [supermerill/SuperSlicer#2641] [Q&A]
Hello all,
New to Superslicer and not a frequent visitor of Github so please excuse my questions.  I have an Ender 3 S1 (not pro version) and am having an issue where the nozzle stops and pauses (and oozes) at the beginning of a new layer.  I have no idea why this is happening and have been trying to look through all the settings to see if there's something amiss.  The part does not have small features that have a short layer print time (I used to use Simplify3D and it had a setting that I think would create a pause if the layer time was very short to allow cooling).  I'm using an Ender 3 profile and Creality PETG and have changed just a few settings from the default but, since I'm unfamiliar, have not changed much.  Can anyone help?
Thanks!

**Comment by supermerill** [comment]
no clue
